tests/integration/api/groups/ListTest.php

```diff
@@ -26,6 +48,35 @@ public function shows_index_for_guest()
         $this->assertEquals(200, $response->getStatusCode());
         $data = json_decode($response->getBody()->getContents(), true);
 
+        $this->assertEquals(Group::where('is_hidden', 0)->count(), count($data['data']));
```

Comment:
Could we improve this (and the other test) to check *which* groups are returned? E.g. map the JSON to an array of IDs and match the one here to `[1]` and the other one to `[1, 10]` (using `assertEqualsCanonicalizing`, see [this StackOverflow answer](https://stackoverflow.com/a/28189403))?